presto-docs/src/main/sphinx/develop/worker-protocol.rst

```diff
@@ -125,3 +125,18 @@ upstream workers using buffer number 2.
 
 .. image:: worker-protocol-output-buffers.png
   :width: 600
+
+Failure Handling
```

Comment:
It would be great to also mention the session / config property you showed me that allows to log all HTTP requests as this is helpful for debugging.

Reply:
Added one more section